### DISCUSSION TITLE
Change in schema identification method

### DISCUSSION THREAD

**Discussion by toshiki-kohno** [django-tenants/django-tenants#920] [Q&A]
In this library, the schema to execute SQL is specified by the tenant name in the url. Is there any way to specify the target schema by the user who logs in, while the url is common to all tenants?

**Comment by tomturner** [comment]
Have a look at this project

https://github.com/Corvia/django-tenant-users

**Comment by toshiki-kohno** [comment]
Thank you for your response.
